tests/special_tests.cpp

```diff
@@ -255,28 +255,28 @@ TEST(special_cases_utf8_utf16_roundtrip) {
     // convert to UTF-16
     size_t utf16words_valid = simdutf::convert_valid_utf8_to_utf16(
         source.c_str(), source.size(), utf16_valid_output.get());
-    ASSERT_TRUE(utf16words_valid == utf16words);
+    ASSERT_EQUAL(utf16words_valid, utf16words);
     for (size_t z = 0; z < utf16words_valid; z++) {
-      ASSERT_TRUE(utf16_valid_output.get()[z] == utf16_output.get()[z]);
+      ASSERT_EQUAL(utf16_valid_output.get()[z], utf16_output.get()[z]);
```

Comment:
Here Visual Studio is unhappy because this involves doing `std::cout << char16_t` and it does not know how to print char16_t.
```suggestion
      ASSERT_EQUAL(uint16_t(utf16_valid_output.get()[z]), uint16_t(utf16_output.get()[z]));
```